### DISCUSSION TITLE
Release Candidate？

### DISCUSSION THREAD

**Discussion by yyliveyy** [dotnet/maui#5007] [Q&A] [answered]
Is RC1 going to be released this month?

**Comment by mcguu** [comment]
I think will release

**Reply by Xyncgas** [reply]
but how do I publish MAUI.blazor (exe? windows store? how to distribute?) how do I change the application's border color
![image](https://user-images.githubusercontent.com/35312807/156901429-64c577af-d207-4a01-a504-2fe1263204bc.png)

**Comment by nogginbox** [comment, marked as answer]
According to the [roadmap](https://github.com/dotnet/maui/wiki/Roadmap) yes, they are going to release R1 in Q1, but I'm not sure this is still going to happen. They seem to be currently focusing on bugs, but the project does not feel in a R1 place yet.

I went to a [virtual talk by David Ortinau](https://www.twitch.tv/videos/1412297653) this week and he did not mention R1, but did talk about going to GA in May.

**Reply by nogginbox** [reply]
Just re-watched the video of the talk and seen in small writing on a slide that the RC planned release date is now April.

**Reply by nogginbox** [reply]
Also found this [Github Planning Board](/orgs/dotnet/projects/85/views/12?layout=board) which may give some insight as to how close they are with RC1.

**Reply by yyliveyy** [reply]
Ok, thank you.It seems that we need patience.
